src/test/util/setup_common.cpp

```diff
@@ -381,16 +381,14 @@ std::pair<CMutableTransaction, CAmount> TestChain100Setup::CreateValidTransactio
     for (const auto& outpoint_to_spend : inputs) {
         // - Use GetCoin to properly populate utxo_to_spend,
         Coin utxo_to_spend;
-        bool has_utxo = coins_cache.GetCoin(outpoint_to_spend, utxo_to_spend);
```

Comment:
This was addressed in https://github.com/bitcoin/bitcoin/pull/30849/files#diff-6a8ef76c60f30a6ca67d9f0e478fd02989c4b7fbc4c3116f80e13d873d5775e6R454 without an assert/Assert